frame/bounties/src/lib.rs

```diff
@@ -475,13 +547,19 @@ decl_module! {
 					BountyStatus::CuratorProposed { ref curator } => {
 						ensure!(signer == *curator, Error::<T>::RequireCurator);
 
+						// Reserve the Curator deposit
 						let deposit = T::BountyCuratorDeposit::get() * bounty.fee;
 						T::Currency::reserve(curator, deposit)?;
 						bounty.curator_deposit = deposit;
 
+						// Reserve the fee for Curator
+						let bounty_account = Self::bounty_account_id(bounty_id);
+						let balance = T::Currency::free_balance(&bounty_account);
+						let fee = bounty.fee.min(balance); // just to be safe
+						T::Currency::reserve(&bounty_account, fee)?;
```

Comment:
why?

Reply:
**Duplicate of design choice reason shared above**

In this context its "accept_curator()" sequence no 4, 5.

![](https://i.imgur.com/y9xwDxh.png)

![](https://i.imgur.com/nQH8cEM.png)

Reply:
As suggested, design for subbounty fund management got changed, Now each subbounty got unique account.
Code is removed.